### ISSUE TITLE
Cannot pass item properties when getting list of items

### ISSUE THREAD

**Issue by Alien-x** [recombee/go-api-client#1]
Hey, I am trying to get list of items from Recombee, but it seems there is some sort of bug in this API causing `includedProperties` value be incorrectly set. The request is:
```
request := recombeeClient.NewListItems().
		SetReturnProperties(true).
		SetIncludedProperties([]string{"type", "description"})
```
Which returns error `Status 400: {\"message\": \"property of items \\\"[type description]\\\" does not match ^[a-zA-Z0-9_\\\\-:@\\\\.]+$\"}`.

**Comment by OndraFiedler** [comment]
Hi @Alien-x,

Thanks for the report.

We have just fixed the issue in `v4.1.1`.

**Comment by Alien-x** [comment]
Thanks for solving it so fast ;)
